webroot/results/includes/events_persons.php

```diff
@@ -8,17 +8,17 @@
     competition.cellName competitionName,
     country.name         countryName
   FROM
-    (SELECT MIN(valueAndId) valueAndId
+    (SELECT MIN(value)
      FROM Concise${valueName}Results result
      WHERE $valueSource>0 $eventCondition $yearCondition $regionCondition
      GROUP BY personId
-     ORDER BY valueAndId
+     ORDER BY value
      $limitCondition) top,
     Results result,
     Competitions competition,
     Countries country
   WHERE " . randomDebug() . "
-    AND result.id      = valueAndId % 1000000000
+    AND result.id      = resultId
```

Comment:
Hmm...I don't think we have access to a resultId here. I can't think of a fix for this, it feels like the same problem we struggled with all day when trying to generate the `Concise*Results` tables: we can get a `MIN(value)`, but we can't also get the corresponding `resultId`